engine/client/client.h

```diff
@@ -844,6 +844,9 @@ movevars_t *pfnGetMoveVars( void );
 
 _inline cl_entity_t *CL_EDICT_NUM( int n )
```

Comment:
Maybe it should not be called when clgame.entities is NULL?

Reply:
Yeah, that's better way

Reply:
Maybe write assert here instead return NULL? Or not assert, but something for to break debugger here